_posts/2018-7-18-EIA-coal.md

```diff
@@ -0,0 +1,139 @@
+---
+
+layout: post
+
+title: "A Visual Exploration of Coal and Electricity Generation"
+
+author: "Chris Davis"
+
+comments: false
+---
+
+<script type="text/javascript" async
+  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML">
+</script>
+
+# Introduction
+
+In the US, discussions regarding coal tend to be [divisive](https://www.economist.com/united-states/2014/05/15/the-politics-of-coal){:target="_blank"}, and are often driven more by politics than data.  In this post, we will take an exploratory dive into detailed public data related to electricity generation from coal, and show how this can be used to better understand these systems and the changes they have been undergoing in recent years.
+
+We will take a look specifically at aspects related to the life cycle of coal, from its origin in coal mines, to its transport within and between regions, and on to its final use in coal-fired power plants.  We will also focus largely on coal use for electricity generation, as the US electric power sector is responsible for about [90% of overall consumption of coal](https://www.eia.gov/coal/production/quarterly/pdf/t32p01p1.pdf){:target="_blank"}.
```

Comment:
For some reason, "transport" reads awkward here to me. Not sure if it should be "transportation". How about "shipment" instead?